libs/common/src/sql-db-entity/utilisation-report.ts

```diff
@@ -0,0 +1,57 @@
+import { Column, Entity, OneToMany, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { UtilisationReportReconciliationStatus } from '../types';
+import { ReportPeriodPartial } from './partials';
+import { AzureFileInfoEntity } from './azure-file-info';
+import { UtilisationDataEntity } from './utilisation-data';
+
+@Entity('UtilisationReport')
+export class UtilisationReportEntity {
```

Comment:
Do we want a @createdAt and @updatedAt column on this as well? The latter would be especially helpful for audit down the line and could set the precedent for them. CreatedAt might be good to audit when the job added it etc. (and date uploaded being the business logic side!)

Reply:
I've added a new abstract `AuditableEntity` class that all our entities can extend.